packages/manager/apps/dedicated/client/app/dedicated/server/ovh-tasks/dedicated-server-ovh-tasks.routes.js

```diff
@@ -3,7 +3,11 @@ angular
   .config(/* @ngInject */($stateProvider) => {
     $stateProvider.state('app.dedicated.server.dashboard.ovh-tasks', {
       url: '/ovh-tasks',
-      component: 'dedicatedServerOVHTasks',
+      views: {
+        'tabView@app.dedicated.server': {
```

Comment:
```suggestion
        'tabView@app.dedicated.server': 'dedicatedServerOVHTasks'
```
is enough ;)

Reply:
Thanks for the suggestion @FredericEspiau !